src/sentry/ownership/grammar.py

```diff
@@ -86,6 +90,35 @@ def load(cls, data):
             data['pattern'],
         )
 
+    def test(self, data):
+        return getattr(self, 'test_%s' % self.type)(data)
+
+    def test_url(self, data):
+        try:
+            url = data['sentry.interfaces.Http']['url']
+        except KeyError:
+            return False
+        return fnmatch(url, self.pattern)
+
+    def test_path(self, data):
+        for frame in _iter_frames(data):
+            try:
+                filename = frame['filename']
+            except KeyError:
+                try:
+                    filename = frame['abs_path']
+                except KeyError:
+                    continue
+
+            # fnmatch keeps it's own internal cache, so
+            # there isn't any optimization we can do here
+            # by using fnmatch.translate before and compiling
+            # our own regex.
+            if fnmatch(filename, self.pattern):
```

Comment:
Will we document somewhere in app that we're using [`fnmatch`](https://docs.python.org/2/library/fnmatch.html) to match the expressions. /cc @MaxBittker